setup.py

```diff
@@ -0,0 +1,2 @@
+import os
+os.system("pip install -r requirements.txt")
```

Comment:
Rather than doing this, I would recommend you look up pyinstaller, or similar libraries to create a binary. As you could image, your customer might not have python installed, or have python 2 installed instead. You wouldn't want them to accidently use that